apps/web/src/components/Settings/Dispatcher/index.tsx

```diff
@@ -45,6 +79,38 @@ const DispatcherSettings: FC = () => {
           </div>
           <ToggleDispatcher />
         </Card>
+        {currentProfile?.dispatcher?.canUseRelay && !idKitData?.isIDKitPhoneVerified && (
+          <Card className="linkify space-y-2 p-5">
+            <div className="flex items-center space-x-2">
+              <div className="text-lg font-bold">Unlock higher dispatcher limits</div>
+            </div>
+            <div className="pb-2">
+              <Trans>By verifying your phone number, we'll relax the gass-less limits.</Trans>
+            </div>
+            <IDKitWidget
+              enableTelemetry
+              methods={['phone', 'orb']}
+              actionId={IDKIT_ACTION_ID}
+              handleVerify={handleIDKitVerify}
+              signal={currentProfile?.ownedBy}
+              copy={{
+                title: 'Lens Dispatcher',
+                heading: t`Verify your phone number to increase gassless limits`
+              }}
```

Comment:
@bigint Since these props are duplicated across two files, what would be the most appropriate place to store a "wrapper component"?